doc/engineering-reference/src/simulation-models-encyclopedic-reference-003/ideal-loads-air-system.tex

```diff
@@ -129,5 +133,21 @@ \subsubsection{Inputs and Data}\label{inputs-and-data-002}
   \item Set the zone inlet node conditions to the supply air mass flow rate, temperature, and humidity ratio.
   \item Calculate the unit output and load components.
   \item If a zone return plenum is used, after all ideal loads systems that are connected to the plenum are simulated, simulate the return plenum.
+  \item Calculate the heating and cooling fuel energy rates from the respective loads and user specified fuel efficiencies:
+
+    \begin{equation}
+      {E_heat} = {\dot Q_load_heat}/(eff_{heat})
+	  {E_cool} = {\dot Q_load_cool}/(eff_{cool})
+    \end{equation}
+	
+    \begin{itemize}
+      \item {E_heat} = total heating fuel energy comsumption rate (W)
+	  \item {E_cool} = total cooling fuel energy comsumption rate (W)
+	  \item {\dot Q_load_heat} = zone total heating load (W)
```

Comment:
These warnings must mean something because building the doc shows some weird formatting. Lines 139 and 140 above are on the same line in the pdf. And the equation definitions are also strange.

<img width="648" height="351" alt="image" src="https://github.com/user-attachments/assets/fbd951d0-eb78-4bd9-9ada-1a0c886a0645" />

Reply:
I will look into it. @rraustad Thanks.